app/helpers/application_helper.rb

```diff
@@ -14,11 +14,11 @@ def navigation
         header.with_action_link(text: 'Sign out', href: destroy_user_session_path, options: { id: 'sign-out-desktop', data: { turbo_method: :get }, inverse: true })
```

Comment:
Can you revert the changes in this file please

Reply:
[773178d](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/773178de60d063b1e23b023cab9470aec05273c7)